package.json

```diff
@@ -136,9 +136,10 @@
     "node": "^14.15.0 || >=16.13.0"
   },
   "dependencies": {
-    "@aws-sdk/client-ec2": "3.112.0",
-    "@aws-sdk/client-ecr": "3.112.0",
-    "@aws-sdk/client-s3": "3.113.0",
+    "@aws-sdk/client-ec2": "3.154.0",
```

Comment:
please don't upgrade packages in this PR

Reply:
The tests I added do not compile if there is a mismatch in the client package versions. I could try using an old version of the `@aws-sdk/client-rds` package in my branch to see if that works.

Reply:
OK, the old version seems to work.